src/main/scala/com/mwz/sonar/scala/scalastyle/ScalastyleRulesRepository.scala

```diff
@@ -32,7 +32,7 @@ import org.sonar.api.server.rule.{RuleParamType, RulesDefinition}
  * Defines a repository for the Scalastyle inspections.
  */
 final class ScalastyleRulesRepository extends RulesDefinition {
-  import ScalastyleRulesRepository._
+  import ScalastyleRulesRepository._ // scalastyle: ignore
```

Comment:
Remember you disable the wildcard import rule, so this is not necessary 😉

Reply:
Well it was mainly an style opinion - When I import something from the companion object in a class, I've always put it as the first line in the class. _(I probably have done that way for [this example](https://docs.scala-lang.org/tour/singleton-objects.html) I read when first learning Scala)_.

So when I was reviewing your code the first time you added that import, I was lost at the beginning, thinking **"how can he use those values without referring to the companion object?!"**, until a few minutes later I saw the import in the header.

Thus, it really doesn't matter, it is only my personal preference - I've searched but can't find any official or external recommendation regarding that specific import.

Anyways, to be honest my personal preference as today is to always prefix those values with the companion object name, instead of importing them. But again, just my taste, nothing else - if you want to change them to avoid _ignoring_ the rule, that's ok :+1:

Reply:
Cool, thanks for the explanation - let's leave this as is and if we ever want to change it we can always come back to it.